pkg/apis/neutron/v1/ovsnnodeosp_types.go

```diff
@@ -0,0 +1,55 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OVSNodeOspSpec defines the desired state of OVSNodeOsp
+// +k8s:openapi-gen=true
+type OVSNodeOspSpec struct {
+	// container image to run for the daemon
+	OvsNodeOspImage string `json:"ovsNodeOspImage"`
+	// service account used to create pods
+	ServiceAccount string `json:"serviceAccount"`
+	// Name of the worker role created for OSP computes
+	RoleName string `json:"roleName"`
+	// log level
+	OvsLogLevel string `json:"ovsLogLevel"`
+	// ovn south db ip
+	OvnSbRemote string `json:"ovnSbRemote"`
```

Comment:
as discussed, this probably should be read from a configmap, instead of being part of the CRD, similarly to the approach @stuggi took for the nova-operator

Reply:
will be done in follow-up patch